### PR TITLE
Add more cool faces 

### PR DESCRIPTION
**Pull request by mabrasil**
I just thought about adding a bunch of more fancy faces I like :smile_cat:

### CODE REVIEW

**Comment by phil-r** [comment]
:+1:

**Comment by bcomnes** [comment]
These are pretty cool.  LGTM.

File: index.js [hunk 1]
```diff
@@ -170,7 +171,107 @@ var faces = [
...
+  "(• ε •)",
+  "[̲̅$̲̅(̲̅ ͡° ͜ʖ ͡°̲̅)̲̅$̲̅]",
+  "| (• ◡•)| (❍ᴥ❍ʋ)",
+  "(◕‿◕✿)",
+  "(╯°□°)╯︵ ʞooqǝɔɐɟ",
+  "(☞ﾟヮﾟ)☞ ☜(ﾟヮﾟ☜)",
+  "(づ￣ ³￣)づ",
+  "(;´༎ຶД༎ຶ`)",
+  "♪~ ᕕ(ᐛ)ᕗ",
+  "༼ つ  ͡° ͜ʖ ͡° ༽つ",
```
Comment (ChennyBaBy) [inline index.js]: Line 56 with hands!

**Comment by ChennyBaBy** [comment]
:+1:

**Pull request merged**